pkg/apis/sources/v1alpha1/azureiothubsource_types.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright (c) 2021 TriggerMesh Inc.
```

Comment:
This should read `Copyright 2021 TriggerMesh Inc.` Additionally, a newline needs to be between the copyright and the rest of the license.